en/docs/administration/production-deployment-guideline.md

```diff
@@ -0,0 +1,72 @@
+# Production Deployment guideline
+
+The requirements for deploying WSO2 products can vary based on the deployment scenario and pattern. The recommendations in this topic are intended for general production use.
```

Comment:
I think we should not specify this. We are governing the patterns that the users can deploy.

Reply:
removed